kore/src/Debug.hs

```diff
@@ -194,8 +192,8 @@ debugPrecAux datatypeInfo (SOP sop) =
     constrs :: NP ConstructorInfo xss
     constrs =
         case datatypeInfo of
-            SOP.ADT     _ _ cs -> cs
-            SOP.Newtype _ _ c  -> c :* Nil
+            SOP.ADT     _ _ cs _ -> cs
+            SOP.Newtype _ _ c    -> c :* Nil
```

Comment:
It would probably be better to change `debugPrecAux` to take only an `NP ConstructorInfo xss` instead of the full `DatatypeInfo`, because we only use one field.